rasterio/mask.py

```diff
@@ -74,10 +74,11 @@ def raster_geometry_mask(dataset, shapes, all_touched=False, invert=False,
         pad_y = 0
 
     north_up = dataset.transform.e <= 0
+    rotated = dataset.transform.b != 0 or dataset.transform.d != 0 
```

Comment:
I do actually calculate whether `rotated` should be true or false based on the dataset transform, just like `north_up`. This could easily be moved into `geometry_window`, however.

Reply:
Yes. Hmm, and good point about changing the API if `north_up` was moved. Let's leave this as it is. I'll approve and merge. Thanks again!

Reply:
Of course! Thanks for the review and the merge.